tensorflow/lite/delegates/vx-delegate/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright 2021 NXP
```

Comment:
Hi Robert,

Can we just use the apache license header? we didn't put our copyright to this project before.

Reply:
Ok, done.